commit/merkleroot/outcome.go

```diff
@@ -245,12 +257,16 @@ func getConsensusObservation(
 			fmt.Errorf("no consensus value for fDestChain, destChain: %d", destChain)
 	}
 
+	// convert aggObs.RMNRemoteConfigs to a map of RMNRemoteConfigs
+	rmnRemoteConfigs := map[cciptypes.ChainSelector][]rmntypes.RMNRemoteConfig{destChain: aggObs.RMNRemoteConfigs}
```

Comment:
I think the plugin only cares about the destination chain remote config.
Maybe no need to keep a map.

Reply:
I see, because you wanted to reuse `GetConsensusMap`

Reply:
I recommend adding some tests to `GetConsensusMap` with this specific map we provide here.

Reply:
Done